src/ooxml/java/org/apache/poi/xslf/usermodel/XSLFRelation.java

```diff
@@ -19,123 +19,132 @@ Licensed to the Apache Software Foundation (ASF) under one or more
-   );
-   
-   public static final XSLFRelation PRESENTATIONML_TEMPLATE = new XSLFRelation(
-           "application/vnd.openxmlformats-officedocument.presentationml.template.main+xml",
-           null, null, null
-   );
-   
-   public static final XSLFRelation PRESENTATION_MACRO = new XSLFRelation(
-           "application/vnd.ms-powerpoint.presentation.macroEnabled.main+xml",
-           null, null, null
-   );
-   
-   public static final XSLFRelation THEME_MANAGER = new XSLFRelation(
-           "application/vnd.openxmlformats-officedocument.themeManager+xml",
-           null, null, null
-   );
-   
-   public static final XSLFRelation NOTES = new XSLFRelation(
-           "application/vnd.openxmlformats-officedocument.presentationml.notesSlide+xml",
-           "http://schemas.openxmlformats.org/officeDocument/2006/relationships/notesSlide", 
-           "/ppt/notesSlides/notesSlide#.xml",
-           XSLFNotes.class
-   );
-   
-   public static final XSLFRelation SLIDE = new XSLFRelation(
-           "application/vnd.openxmlformats-officedocument.presentationml.slide+xml",
-           "http://schemas.openxmlformats.org/officeDocument/2006/relationships/slide", 
-           "/ppt/slides/slide#.xml", 
-           XSLFSlide.class
-   );
-   
-   public static final XSLFRelation SLIDE_LAYOUT = new XSLFRelation(
-         "application/vnd.openxmlformats-officedocument.presentationml.slideLayout+xml",
-         "http://schemas.openxmlformats.org/officeDocument/2006/relationships/slideLayout",
-         "/ppt/slideLayouts/slideLayout#.xml", 
-         XSLFSlideLayout.class
-   );
-   
-   public static final XSLFRelation SLIDE_MASTER = new XSLFRelation(
-         "application/vnd.openxmlformats-officedocument.presentationml.slideMaster+xml",
-         "http://schemas.openxmlformats.org/officeDocument/2006/relationships/slideMaster",
-         "/ppt/slideMasters/slideMaster#.xml",
-         XSLFSlideMaster.class
-   );
-
-   public static final XSLFRelation NOTES_MASTER = new XSLFRelation(
-         "application/vnd.openxmlformats-officedocument.presentationml.notesMaster+xml",
-         "http://schemas.openxmlformats.org/officeDocument/2006/relationships/notesMaster",
-         "/ppt/notesMasters/notesMaster#.xml",
-         XSLFNotesMaster.class
-   );
-
-   public static final XSLFRelation COMMENTS = new XSLFRelation(
-         "application/vnd.openxmlformats-officedocument.presentationml.comments+xml",
-         "http://schemas.openxmlformats.org/officeDocument/2006/relationships/comments",
-         "/ppt/comments/comment#.xml",
-         XSLFComments.class
-   );
-   
-   public static final XSLFRelation COMMENT_AUTHORS = new XSLFRelation(
-         "application/vnd.openxmlformats-officedocument.presentationml.commentAuthors+xml",
-         "http://schemas.openxmlformats.org/officeDocument/2006/relationships/commentAuthors",
-         "/ppt/commentAuthors.xml",
-         XSLFCommentAuthors.class
-   );
-   
+    /**
```

Comment:
Seems like your editor changed all space-only indentation to some convention specific to your company.